project.yaml

```diff
@@ -45,9 +45,31 @@ actions:
       moderately_sensitive:  
         figs: output/figs.svg
 
+  generate_study_population_1:
+    run: cohortextractor:latest generate_cohort --study-definition study_definition_monthly --index-date-range "2020-03-01 to 2021-02-01 by month" --output-format feather
+    outputs:
+      highly_sensitive:
+        cohort: output/monthly/input_monthly_*.feather
+
+  generate_study_population_2:
```

Comment:
Just to confirm, are these being split up to avoid memory problems?

Reply:
Yea, that's right. Are you happy with 2020-03-01 as the start date? I will extend it up to May 2022.